kimhaejung/app.js

```diff
@@ -52,6 +56,59 @@ app.post("/users/signup", async (req, res, next) => {
   res.status(201).json({ message: "userCreated" });
 });
 
-app.listen(port, function () {
-  console.log(`server listening on port ${port}`);
+//
+app.patch("/posts/:userId/:postId", async (req, res, next) => {
+  const { userId, postId } = req.params;
+  const { content } = req.body;
+  await appDataSource.query(
+    ` UPDATE 
+    posts 
+    SET content=? 
+    WHERE user_id=${userId} AND id=${postId};
+    `,
+    [content]
+  );
```

Comment:
- 가독성을 위해서 아래와 같이 바꾸어 보는 것은 어떨까요?

```suggestion
  await appDataSource.query(` 
  UPDATE 
    posts 
  SET content=? 
  WHERE user_id=${userId} AND id=${postId};
    `,
    [content]
  );
```

Reply:
- 또한 해당 변수에 악의적인 SQL 문을 심는 injection 공격을 고려하여, ? 물음표 placeholder를 사용하여 해당 내용을 1차로 차단하는 효과를 얻어낼 수 있습니다.
- content에서 해주신 내용처럼 userId와 postId를 placeholder로 치환하여 사용할 수 있게 수정해주세요.